src/auth/facebook.js

```diff
@@ -0,0 +1,95 @@
+import passport from "passport";
+import FacebookStrategy from "passport-facebook";
+import { isNil, kebabCase } from "lodash";
+import { app, driver, SECRET } from "../index";
+import { createToken } from "./auth";
+
+// Setting up our Facebook strategy for passport
+passport.use(
+  new FacebookStrategy(
+    {
+      clientID: process.env.FB_ID, // found on your facebook app profile
+      clientSecret: process.env.FB_SECRET, // found on your facebook app profile
+      callbackURL:
+        process.env.FB_CALLBACK_URI ||
+        "http://localhost:8000/login-facebook/callback", // should be the base URI for your server followed `/login-facebook/callback`
```

Comment:
We can replace this with simply `/login-facebook/callback` if in-case someone wants to use a different PORT for their local system.

Reply:
thanks, I will do